Documentation/contributing/testing/e2e.rst

```diff
@@ -503,9 +503,9 @@ cluster.
 2- Invoke the tests from ``cilium/test`` with options set as explained in
 `Running End-To-End Tests In Other Environments via kubeconfig`_
 
-.. note:: GKE clusters may have namespaces stuck at the ``Terminating`` state,
-          causing Ginkgo tests to fail. If so, you'll see them in ``kubectl get ns``,
-          and can get rid of them by running ``cilium/test/gke/clean-cluster.sh``.
+.. note:: The tests require the ``NATIVE_CIDR`` environment variable to be set to
+          the value of the cluster IPv4 CIDR returned by the ``gcloud container
+          clusters describe`` command.
```

Comment:
The original commit only deleted, not sure why this got added in.

Reply:
Will delete, I guess I messed up the conflict resolution.